### PR TITLE
Taking column case sensitivity into consideration when migrating a MS…

### LINKED ISSUE #737: Migrating a MS SQL database to SQLite doesn't take column case sensitivity into consideration
When I migrate a MS SQL database to SQLite, the generated schema doesn't take column case sensitivity into consideration.

### Steps to reproduce
1. Create a MS SQL database using a case insensitive collation (ex. "SQL_Latin1_General_CP1_CI_AS").
2. Create a database table with a string based column using a case insensitive collation (ex. "SQL_Latin1_General_CP1_CI_AS").
3. Migrate the database to SQLite
4. Notice the generated column is case sensitive.

Here's some code to determine whether a column is case sensitive or not:
https://github.com/jschultz/sqlconverter/blob/7f151264aa5931da2e759dc6a2dcb7ec2310f295/DbAccess/SqlServerToSQLite.cs#L1031-L1062

Thank you.

### Further technical details
Toolbox/Power Tools version: 4.7.577

Database engine: SQL Server 2014 --> SQlite

Visual Studio or SSMS version: Visual Studio 2015 Update 3

### PR DESCRIPTION
**Pull request by icnocop**
… SQL database to SQLite

Fixes #737

### CODE REVIEW

File: src/API/Generator/Generator.cs [hunk 1]
```diff
@@ -1662,19 +1662,21 @@ private string GenerateColumLine(bool includeData, Column col, bool azure)
...
+            string collate = !col.IsCaseSensitivite ? string.Empty : " COLLATE NOCASE";
             switch (col.DataType)
             {
                 case "nvarchar":
                 case "nchar":
                 case "binary":
                 case "varbinary":
                     line = string.Format(CultureInfo.InvariantCulture,
-                        "[{0}] {1}({2}) {3} {4}"
+                        "[{0}] {1}({2}) {3} {4}{5}"
```
Comment (ErikEJ) [inline src/API/Generator/Generator.cs]: Only applies if exporting to SQLite...

**Comment by ErikEJ** [comment]
Please make this fix apply to SQLite only, otherwise LGTM

**Comment by icnocop** [comment]
Done.

I also fixed a minor issue with extra spaces when generating the column lines.

Thank you.

**Comment by ErikEJ** [comment]
Perfect, thanks

**Pull request merged**